gfx/Mushroom-Dream/pngs_tall_32x48/furniture/t_console.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "id": "t_console",
+    "fg": "t_console",
+    "bg": ""
```

Comment:
```suggestion
    "bg": [
    { "weight": 2, "sprite": "t_floor_1" },
    { "weight": 2, "sprite": "t_floor_2" },
	  { "weight": 2, "sprite": "t_floor_4" },
    { "weight": 1, "sprite": "t_floor_3" }
  ]
```
Because otherwise the background of the console will just be black since it's not a furniture but terrain. I'm not sure putting the 32x32 floor will work like this maybe you'd need to make a 32x48 version of the floor with the sprite at the bottom and transparency in the top part.

Reply:
Hey fris! I actually got the tile console to draw properly even with no background. See this screenshot
![image](https://user-images.githubusercontent.com/80914993/114306696-8d578700-9ad4-11eb-86bc-a02ab35d4500.png)

Even if you hit and destroy the console the background works fine for some reason 

![image](https://user-images.githubusercontent.com/80914993/114306738-bb3ccb80-9ad4-11eb-9a4b-0c98f7028f97.png)

Reply:
That's weird but ok if it already works then it's fine I guess